packages/editor/src/components/post-sync-status/index.js

```diff
@@ -109,6 +109,7 @@ export function PostSyncStatusModal() {
 						<VStack spacing="5">
 							<ReusableBlocksRenameHint />
 							<ToggleControl
+								// translators: Toggle button label, when active makes an individual pattern "Synced".
```

Comment:
I was just alterted to this - https://core.trac.wordpress.org/ticket/34753

So I'll change the use of 'toggle' in this translator comment.